### PR TITLE
Xxdiff is a Graphical merge tool

### PR DESCRIPTION
**Pull request by robertdfrench**
Folks who still use CVS (folks still use CVS) seem to enjoy this tool.

### CODE REVIEW

**Comment by citibeth** [comment]
Is there any downside for this being a Python extension?  You don't have to say `spack activate` if you don't want to...

Even if it's not a Python extension, I woud support renaming it to `xxdiff`.  The `py-` notation should be reserved for libraries, IMHO.

**Comment by adamjstewart** [comment]
@citibeth I agree. There's really no downside to making it a Python extension. The only difference between an extension and a depends_on (in my mind) is that you can activate it. If it builds a site-packages directory, I think it's worth keeping it an extension.

I just tried building this package. It builds perfectly fine and does indeed create a site-packages directory. So it should probably extend Python, but we should maybe rename it to xxdiff since other packages aren't likely to depend on it. But like I said, I'm fine either way.

**Comment by tgamblin** [comment]
@robertdfrench: can you change the name to xxdiff?  I don't think this is "pythony" enough to be a `py-*` package.

**Comment by robertdfrench** [comment]
Agreed. Will fix.

File: var/spack/repos/builtin/packages/xxdiff/package.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+class Xxdiff(Package):
+    """xxdiff: Graphical File And Directories Comparator And Merge Tool"""
+
+    homepage = "https://bitbucket.org/blais/xxdiff"
+    url      = "https://sourceforge.net/projects/xxdiff/files/xxdiff/4.0.1/xxdiff-4.0.1.tar.bz2/download"
+
+    version('4.0.1', '34253b829e249faf2480a989487e99bc')
+
+    depends_on('python')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/xxdiff/package.py]: Can you change this back to `extends()`? The package does produce a `site-packages` directory, so I think it makes sense to extend Python.

**Comment by citibeth** [comment]
I don't think this is yet ready to merge.  The xxdiff documentation says this package has dependencies, none of which seem to be listed in the Spack package.  From the xxdiff webpage:

> 
> Dependencies
> 
> Build dependencies are:
> 
>     Qt-4.5 or higher
>     a C++ compiler, flex and yacc
> 
> Runtime dependencies are:
> 
>     Qt-4.5 or higher
>

**Comment by robertdfrench** [comment]
omfg is it picking up system qt and I didn't realize? Will fix. Thanks
@citibeth

**Pull request closed without merging**